.travis.yml

```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "4"
```

Comment:
Just to be clear, needing to change from Node 4 to another version is _generally_ a breaking change (since it means we don't support Node 4 any longer).

Why do we need this change as part of this PR?

Reply:
There was some dependency failure which was not supported in node 4. Now I have upgraded ember-cli with the latest commit. That will take care of this.